### PR TITLE
Update R23 roadmap

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Update R23 roadmap to account for items found in Q1 of this board: https://miro.com/app/board/uXjVOdo-7uU=/.

### CODE REVIEW

File: src/data/doc-it/R23Roadmap.ts [hunk 1]
```diff
@@ -85,49 +85,9 @@ export const roadmap: RoadmapBucket[] = [
...
-    {
-        name: 'Themes',
-        description: 'Styles and colors for applications',
-        type: 'development',
-        framework: ['all'],
-        items: [ 
             {
-                name: 'Accessible Themes',
-                description: 'Exploration of more accessible themes',
+                name: 'Mobile Charting Guidelines',
```
Comment (huayunh) [inline src/data/doc-it/R23Roadmap.ts:89]: Ambitious... a draft maybe.

**Review by huayunh — COMMENTED** [review]
The table guideline counts as a Q1 item and is sorta finished. May want to add an item about that and mark it complete.

File: src/data/doc-it/R23Roadmap.ts (hunk 1 above)
Comment (emclaug2) [reply src/data/doc-it/R23Roadmap.ts:89]: @joebochill Can you clarify what's in the "Mobile Charting" item in the roadmap, was this expanded on in the backlog?

Comment (joebochill) [reply src/data/doc-it/R23Roadmap.ts:89]: We can call it an exploration/evaluation of mobile charting libraries

File: deployed/doc-it/R23Roadmap.json [hunk 2]
```diff
@@ -129,7 +83,7 @@
     ],
     "items": [
       {
-        "name": "Training Materials",
+        "name": "Bootcamp Training Materials",
```
Comment (joebochill) [inline deployed/doc-it/R23Roadmap.json:86]: This is not going to be out in Q1...more likely late Q2 or Q3. I would leave it off here (even though we're going to be doing a lot of pre-work now)

**Review by joebochill — COMMENTED** [review]
Can you add a high-level R24 file as well so people can see what's on the plan for next quarter too?

File: src/data/doc-it/R23Roadmap.ts (hunk 1 above)
Comment (emclaug2) [reply src/data/doc-it/R23Roadmap.ts]: Sounds good, this description has been updated to be 'exploratory'.

File: deployed/doc-it/R23Roadmap.json (hunk 2 above)
Comment (emclaug2) [reply deployed/doc-it/R23Roadmap.json]: Ok I've moved this to the R24 roadmap.

File: deployed/doc-it/R24Roadmap.json [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        "author": "Brightlayer UI"
+      },
+      {
+        "name": "Windows Support",
+        "description": "Making sure all Brightlayer UI resources run seamlessly on both Mac and Windows devices ",
+        "year": 2022,
+        "quarter": "Q2",
+        "status": "backlog",
+        "author": "Brightlayer UI"
+      },
```
Comment (joebochill) [inline deployed/doc-it/R24Roadmap.json:117]: I would skip this one too since it's an internal thing for us

File: deployed/doc-it/R24Roadmap.json [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      {
+        "name": "Iconography Guidelines",
+        "description": "Refactoring our icons page and updating guidelines on using icons",
```
Comment (joebochill) [inline deployed/doc-it/R24Roadmap.json:120]: This one is in here twice

File: deployed/doc-it/R24Roadmap.json [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        "year": 2022,
+        "quarter": "Q2",
+        "status": "backlog",
+        "author": "Brightlayer UI"
+      },
+      {
+        "name": "Component Library Sandbox and Documentation",
```
Comment (joebochill) [inline deployed/doc-it/R24Roadmap.json:127]: This one is also in here twice...recommend 0 times

File: deployed/doc-it/R24Roadmap.json [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+        "description": "Explore Storybook alternatives; begin development towards a new component library sandbox",
+        "year": 2022,
+        "quarter": "Q2",
+        "status": "backlog",
+        "author": "Brightlayer UI"
```
[the remark this answers is not in the document]
Comment (emclaug2) [reply deployed/doc-it/R24Roadmap.json:72]: This is removed for now.

File: deployed/doc-it/R24Roadmap.json (hunk 3 above)
Comment (emclaug2) [reply deployed/doc-it/R24Roadmap.json:117]: Yanked this one out.

File: deployed/doc-it/R24Roadmap.json (hunk 4 above)
Comment (emclaug2) [reply deployed/doc-it/R24Roadmap.json:120]: Thanks, good catch.

**Comment by emclaug2** [comment]
Ok this has been updated; all comments should be addressed at this point.

**Review by joebochill — APPROVED** [review]

**Pull request merged**